app/components/layer-advanced/template.hbs

```diff
@@ -1,5 +1,12 @@
 {{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"}}
 
-<div id="froala-editor"  class="fr-view">
-</div>
+<!-- <div id="froala-editor"  class="fr-view">
+</div> -->
 
+{{#if editMode}}
+	{{quill-editor value=layer.content options=options textChange=(action "updateText")}}
```

Comment:
Please add margins to top and bottom and lighten border color, a suggestion:

<img width="895" alt="screen shot 2017-07-31 at 2 17 00 pm" src="https://user-images.githubusercontent.com/1314003/28791612-08361ad4-75fb-11e7-8ed0-4a13bcf78a68.png">